extensions/amp-bodymovin-animation/0.1/test/test-amp-bodymovin-animation.js

```diff
@@ -0,0 +1,74 @@
+
+  function getAnimation(loop) {
+    env.fetchMock.getOnce(jsonLink, {'body': mockResponse});
+
+    const anim = doc.createElement('amp-bodymovin-animation');
+    if (loop) {
+      anim.setAttribute('loop', loop);
+    }
+    doc.body.appendChild(anim);
+    return anim.build().then(() => {
```

Comment:
I keep getting the following error: 

`TypeError: anim.build is not a function`
Am I missing something? There is a buildCallback, I checked a few APIs but none of them are doing much different from what I am.